### PR TITLE
Adds min-width mixin, and maps all values to allow prefixed spacing values

### PR DESCRIPTION
**Pull request by andycarrell**

### CODE REVIEW

File: app/pages/angelfish.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
       <div className="flex-1 w-full">
         <section className="padding-y-4xl">
           <div className="page-container">
-            <div className="flex flex-row justify-content-between mb-6 md:mb-8">
+            <div className="flex flex-row justify-content-between margin-bottom-l m:margin-bottom-4xl">
```
Comment (andycarrell) [inline app/pages/angelfish.tsx:36]: @finnclark not sure of exact values, LMK what you think should go here 🙏

**Comment by finnclark** [comment]
Looks like this hasn't pulled in the latest dropdown styling yet

File: app/styles/spacing.scss [hunk 2]
```diff
@@ -14,74 +14,112 @@ $space-map: (
...
-    margin-bottom: $space;
-  }
-  .margin-x-#{$size} {
-    margin-left: $space;
-    margin-right: $space;
-  }
+@each $prefix, $breakpoint in $breakpoint-map {
+  @each $size, $space in $space-map {
+    // Margin
+    .#{$prefix}margin-#{$size} {
```
Comment (finnclark) [inline app/styles/spacing.scss:21]: Small suggestion, could you please comment in here an example of using these?

File: app/pages/angelfish.tsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
       <div className="flex-1 w-full">
         <section className="padding-y-4xl">
```
Comment (finnclark) [inline app/pages/angelfish.tsx:34]:
```suggestion
        <section className="padding-top-4xl padding-bottom-xl">
```

**Comment by andycarrell** [comment]
@finnclark refreshed!

**Review by finnclark — APPROVED** [review]

**Comment by finnclark** [comment]
1 code suggestion :)

File: app/styles/vars.scss [hunk 4]
```diff
@@ -5,6 +5,26 @@ $breakpoint-s: 640px;
 $breakpoint-m: 768px;
 $breakpoint-l: 1024px;
 
+// map of 'predixes' to equivalent breakpoint
```
Comment (andycarrell) [inline app/styles/vars.scss:8]: lol

```suggestion
// map of 'prefixes' to equivalent breakpoint
```

**Comment by finnclark** [comment]
LGTM

**Pull request merged**